apiserver/facades/client/bundle/bundle.go

```diff
@@ -622,10 +621,19 @@ func (b *BundleAPI) endpointBindings(bindings map[string]string) (map[string]str
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	names := set.NewStrings()
 	for k, v := range endpointsWithSpaceNames {
-		if v == network.DefaultSpaceName {
+		if v == "" {
```

Comment:
As per yesterdays conversation, I believe that values can never be `""`. I think this should read `network.AlphaSpaceName`.

Reply:
this code no long exists after changing in determination for printing bindings.